Typezor.Tests/CodeModel/MethodTests.cs

```diff
@@ -24,7 +31,7 @@ public void Expect_name_to_match_method_name()
 
             methodInfo.Name.ShouldEqual("Method");
             methodInfo.FullName.ShouldEqual("Typezor.Tests.CodeModel.Support.MethodInfo.Method");
-            methodInfo.Parent.ShouldEqual(classInfo);
+            //methodInfo.Parent.ShouldEqual(classInfo);
```

Comment:
May be missed to remove this commented code?

Reply:
fixed